ts/screens/wallet/AddCardScreen.tsx

```diff
@@ -61,6 +58,10 @@ type State = Readonly<{
   expirationDate: Option<string>;
   securityCode: Option<string>;
   holder: Option<string>;
+  isValidHolder?: boolean;
```

Comment:
I suggest to remove these intermediate values from the state as you can derive all of them from the attributes that are already in the state